library/src/main/java/com/airbnb/android/airmapview/WebViewMapFragment.java

```diff
@@ -218,11 +220,19 @@ public void setOnInfoWindowClickListener(OnInfoWindowClickListener listener) {
     infoWindowCreator = creator;
   }
 
+  @Override
   public void getMapScreenBounds(OnMapBoundsCallback callback) {
     onMapBoundsCallback = callback;
     webView.loadUrl("javascript:getBounds();");
   }
 
+  @Override
+  public void getLatLngScreenLocation(LatLng latLng, OnLatLngScreenLocationCallback callback) {
+    onLatLngScreenLocationCallback = callback;
```

Comment:
wound't it be better to set the callback via a dedicated method like `setOnLatLngScreenLocationCallback` and have this method just take the `latLng` and let the callback be fired when it's finished? This would match the way other callbacks are being set